binding/python/test_utils.py

```diff
@@ -0,0 +1,143 @@
+#
```

Comment:
do you need all functions in this file? If not, we can make it shorter and even perhaps combine it with the test file